docs-sources/includes/_Modules.md

```diff
@@ -308,3 +308,68 @@ instead of relying on MAGE to serialize the return data. This can be useful when
 will serve certain pieces of data which will not change frequently. When you wish to do so, simply make
 sure that `exports.serialize` is set to `false`, and to manually return a stringified version of your
 data.
+
+## Error management
+
+> lib/modules/players/usercommands/register.js
+
+```javascript
+'use strict';
+
+const {
+  players,
+  MageError
+} = require('mage');
+
+class NoHomersError extends MageError {
+  constructor(data) {
+		super(data);
+
+    // Code to return to the client (default: server)
+    this.code = 'server';
+
+    // Log level to use to log this error (default: error)
+    this.level = 'warning';
+
+    // Details to log alongside the message (default: undefined)
+    this.details = data.details;
```

Comment:
`details` in the logger used to be like a "long message". Here I guess it's a dictionary (from looking at the code example below)?

Reply:
Fine as-is 👍